builder/llpcBuilderImplSubgroup.cpp

```diff
@@ -1476,11 +1476,11 @@ Value* BuilderImplSubgroup::CreateGroupBallot(
     // The not equal predicate for the icmp intrinsic is 33.
     Constant* const pPredicateNE = getInt32(33);
 
-    SmallVector<Type*, 2> types;
-
     // icmp has a new signature (requiring the return type as the first type).
-    types.push_back(getIntNTy(GetShaderSubgroupSize()));
-    types.push_back(getInt32Ty());
+    Type* types[] = {
+        getIntNTy(GetShaderSubgroupSize()),
+        getInt32Ty()
+    };
 
     Value* pResult = CreateIntrinsic(Intrinsic::amdgcn_icmp,
                                      types,
```

Comment:
Would it be possible to insert the elements to this function call direct without making a temp variable

```
CreateIntrinsic(Intrinsic::amdgcn_icmp,
                        {
                            getIntNTy(GetShaderSubgroupSize()),
                            getInt32Ty()
                        }
```